### ISSUE TITLE
Import organization

### ISSUE THREAD

**Issue by vintage** [Fintasys/emoji_picker_flutter#5]
Would be great to organise the code in such way that app needs to perform one import instead of multiples. Right now I need to:

```
import 'package:emoji_picker_flutter/category_icons.dart';
import 'package:emoji_picker_flutter/config.dart';
import 'package:emoji_picker_flutter/emoji_picker_flutter.dart';
```

would rather see it like:

```
import 'package:emoji_picker_flutter/emoji_picker.dart';
```

which exports all public parts of the library.

**Comment by Fintasys** [comment]
@vintage Ah I see, great suggestion! Will change it in the next days 🙇

**Comment by Fintasys** [comment]
@vintage Fixed in v1.0.3! Thanks again 🙇
